client/src/components/Header/Menu.tsx

```diff
@@ -1,4 +1,6 @@
+/** @jsxImportSource @emotion/react */
```

Comment:
This is necessary due to CRA. I'm not sure if it can be configured globally. See https://emotion.sh/docs/css-prop

> If you are using a zero-config tool with automatic detection of which runtime (classic vs. automatic) should be used and you are already using a React version that has [the new JSX runtimes](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) (hence runtime: 'automatic' being configured automatically for you) such as Create React App 4 then /** @jsx jsx */ pragma might not work and you should use /** @jsxImportSource @emotion/react */ instead.